mayday/mayday.go

```diff
@@ -1,20 +1,42 @@
 package mayday
 
 import (
-	"fmt"
 	"log"
-	"os"
-	"path"
 )
 
-func maybeCreateLink(src, dst, workspace string) error {
-	if src == "" {
-		return nil
+func Run(t Tar, files []File, commands []Command, journals []Journal) error {
+	defer t.Close()
+
+	// copy files and run commands
+	for i := 0; i < len(files); i++ {
```

Comment:
These loops could be written without any index variable by using range, something like:

```
for _, file := range files {
   ...
   t.Add(file.Content, file.Header)
}
```